### PR TITLE
fix: improve prettier message

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

fix

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
No

**Summary**
- improve error message when prettier is not found

**Does this PR introduce a breaking change?**
no

**Other information**
Fix https://github.com/webpack/webpack-cli/issues/2253

### CODE REVIEW

File: packages/generators/src/utils/run-prettier.ts [hunk 1]
```diff
@@ -21,9 +21,7 @@ export function runPrettier(outputPath: string, source: string): void {
         // eslint-disable-next-line node/no-extraneous-require
         prettier = require('prettier');
     } catch (err) {
-        logger.warn(
-            "File is not properly formatted because you don't have prettier installed, you can either install it or format it manually",
-        );
+        logger.warn(`${fileName} is not formatted because prettier is not installed locally.`);
```
Comment (alexander-akait) [inline packages/generators/src/utils/run-prettier.ts:24]: Prettier can be installed globally :smile:

Comment (anshumanv) [reply packages/generators/src/utils/run-prettier.ts:24]: Tried it but for some reason it's not doing a global lookup 🤔 

image

Comment (alexander-akait) [reply packages/generators/src/utils/run-prettier.ts:24]: hm, yep, we need other module for this, maybe we can improve it in future

**Review by alexander-akait — DISMISSED** [review]

**Comment by alexander-akait** [comment]
Let's fix tests

**Comment by anshumanv** [comment]
yep wip

**Comment by anshumanv** [comment]
CI should be green now

**Comment by anshumanv** [comment]
not sure what's wrong with CI, keeps failing

**Comment by rishabh3112** [comment]
> not sure what's wrong with CI, keeps failing

same mjs tests are failing here as well #2423.

**Comment by alexander-akait** [comment]
@anshumanv Please rebase again

**Comment by anshumanv** [comment]
@alexander-akait done, will create branch on upstream from now

**Review by snitin315 — APPROVED** [review]

**Review by alexander-akait — APPROVED** [review]

**Review by jamesgeorge007 — APPROVED** [review]

**Pull request merged**